Podfile

```diff
@@ -5,6 +5,8 @@ yajlVersion='~> 2.0.4'
 
 target 'AdblockPlusSafari' do
 
+pod 'yajl', $yajlVersion
```

Comment:
Why not add it when you need it then? How I understood it, all the other changes here are triggered by a new XCode and/or CocoaPods version, that's why I wanted to have it split out. A PR should contain only the changes needed to solve the issue they're trying to solve, nothing else.